public/app/features/explore/Wrapper.test.tsx

```diff
@@ -117,7 +118,7 @@ describe('Wrapper', () => {
     );
 
     // Editor renders the new query
-    await screen.findByText(`loki Editor input: other query`);
+    await screen.findByText(`elastic Editor input: other query`);
```

Comment:
No idea how this worked before. I think because before there were more rerenders the editor did not change to new editor before this was run and so this just tested whether the old editor was still rendered.